packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -86,7 +86,7 @@ const Layout = () => {
   const searchButtonClickHandler = () => {
     useUIStore.setState({
       searchActive: true,
-      preSearchPage: location,
+      preSearchPage: `${location.pathname}${location.search}`,
```

Comment:
Just a small check, are we sure that the `pathname` and `search` properties are always set?